cms/templates/export.html

```diff
@@ -6,18 +6,25 @@
   from django.utils.translation import ugettext as _
   import json
 %>
-<%block name="title">${_("Course Export")}</%block>
+<%block name="title">
+%if library:
```

Comment:
@sarina Right, thanks. I am referring to a bunch of lines in this HTML file (as well as import.html). For example, lines 11-13, 39-41, 56 & 62, 73-75, 83-86.

An example with both singular and plural is line 56.

Do we have guidelines about how to handle capitalized strings (sometimes we want "library" and sometimes "Library")?